### ISSUE TITLE
Add content to JSON Schema title and description fields in template field

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-template-server#14]
Currently template fields have empty title and description fields, e.g.,

```   
"sampleDescription": {
      "@type": "https://schema.metadatacenter.org/core/TemplateField",
      "@context": {
        "pav": "http://purl.org/pav/",
        "cedar": "https://schema.metadatacenter.org/core/"
      },
      "type": "object",
      "title": "h",
      "description": "h",
```

**Comment by martinjoconnor** [comment]
Will be addressed by #15.

**Comment by kristoole** [comment]
unsubscribe
